assets/src/components/form/model/select/test/datetime.js

```diff
@@ -18,24 +18,13 @@ describe( 'DatetimeSelect()', () => {
 		const wrapper = shallow( <DatetimeSelect /> );
 		expect( wrapper.prop( 'queryData' ) )
 			.toEqual( expectedDefaultQueryData );
-		expect( wrapper.state( 'queryData' ) )
-			.toEqual( expectedDefaultQueryData );
 		expect( wrapper
 			.prop( 'selectConfiguration' ).placeholder ).toEqual(
 			'Select Datetime...',
 		);
 		expect( wrapper.prop( 'modelName' ) ).toEqual( 'datetime' );
 		expect( wrapper.prop( 'label' ) ).toEqual( 'Select Datetime' );
 	} );
-	it( 'matches expected update to queryData state with forEventId prop',
-		() => {
-			const wrapper = shallow( <DatetimeSelect forEventId={ 20 } /> );
-			expect( wrapper.state( 'queryData' ) ).toEqual( {
-				...expectedDefaultQueryData,
-				forEventId: 20,
-			} );
-		},
-	);
```

Comment:
there is no longer any state in the concrete model-selects hence the removal of this test.